specs/protocol/bridges.md

```diff
@@ -6,6 +6,7 @@
 
 - [Overview](#overview)
 - [Token Depositing](#token-depositing)
+- [ERC20 Unlocking](#erc20-unlocking)
```

Comment:
We use to have this built directly into the standard bridge, see https://github.com/ethereum-optimism/optimism-legacy/blob/8205f678b7b4ac4625c2afe351b9c82ffaa2e795/packages/contracts/contracts/L2/messaging/L2StandardBridge.sol#L163

Reply:
this is great! this was our first approach to try to fix the problem.
but we thought that if we were to modify the core contracts, we would rather see a future-proof feature being added than a patch for a specific use-case.
while also keeping at a minimum the required modifications to the current codebase.